pkg/api/validation/validation.go

```diff
@@ -1614,6 +1615,18 @@ func ValidateVolumeMounts(mounts []api.VolumeMount, volumes sets.String, fldPath
 		if mountpoints.Has(mnt.MountPath) {
 			allErrs = append(allErrs, field.Invalid(idxPath.Child("mountPath"), mnt.MountPath, "must be unique"))
 		}
+		// check for subpath conflict
+		for _, apath := range mountpoints.List() {
```

Comment:
This has nothing to do with subPaths right? Just if 2 different mountPaths overlap. since `subPath` are separate field, I would perhaps reword this comment.